src/cljs/main/broadfcui/common/table/table.cljs

```diff
@@ -0,0 +1,141 @@
+(ns broadfcui.common.table.table
+  (:require
+    [clojure.set :refer [difference subset?]]
+    [dmohs.react :as react]
+    [broadfcui.common.components :as comps]
+    [broadfcui.common.table.body :as body]
+    [broadfcui.common.table.column-editor :refer [ColumnEditButton]]
+    [broadfcui.common.table.paginator :as paginator]
+    [broadfcui.common.table.utils :as table-utils]
+    [broadfcui.common.style :as style]
+    [broadfcui.persistence :as persistence]
+    [broadfcui.utils :as utils]
+    ))
+
+
+;; Define default props this way because we need to do a deep-merge,
+;; instead of React's regular merge.
+(def ^:private default-props
+  {:body {:empty-message "There are no rows to display."
+          :external-query-params #{}
+          :behavior {:reorderable-columns? true
+                     :fixed-column-count 0
+                     :sortable-columns? true
+                     :allow-no-sort? false
+                     :resizable-columns? true
+                     :filterable? true}}
+   :toolbar {:style {:display "flex" :alignItems "baseline" :marginBottom "1rem"}
```

Comment:
This base style, maybe not.  But the individual components, yes.  Check out how the library table gets the column reorder button to be on the right.